security_fr.md

```diff
@@ -162,10 +162,11 @@ systemctl restart ssh
 
 ### Désactivation de l’API YunoHost
 
-YunoHost est administrable via une **API HTTP**, servie sur le port 6787 par défaut. Elle permet d’administrer une grande partie de votre serveur, et peut donc être utilisée à des **fins malveillantes**. La meilleure chose à faire si vous êtes habitués aux lignes de commande est de désactiver le service `yunohost-api`, et **utiliser la [ligne de commande](/commandline_fr)** en SSH.
+YunoHost est administrable via une **API HTTP**, servie sur le port 6787 par défaut (seulement sur `localhost`). Elle permet d’administrer une grande partie de votre serveur, et peut donc être utilisée à des **fins malveillantes**. La meilleure chose à faire si vous êtes habitués aux lignes de commande est de désactiver le service `yunohost-api`, et **utiliser la [ligne de commande](/commandline_fr)** en SSH.
 
 ```bash
-sudo service yunohost-api stop
+sudo systemctl disable yunohost-api
```

Comment:
```suggestion
sudo systemctl mask yunohost-api
```
`disable` action doesn't prevent to start service by hand.

Reply:
In some case yunohost restart yunohost-api, if we mask it, is there a risk that the command fails ?

Reply:
Using `mask` will prevent all restart of this service.